builder/README-containers.md

```diff
@@ -0,0 +1,79 @@
+## Instructions on how to use the 'tracee-make' docker container
```

Comment:
I wanted README.md inside builder because when you're navigating github, you click in the "builder" directory and get instructions on how to build. I want to update the official documentation, though, to include information from this README* files there. Maybe to do a new PR to make the builder/ directory README simple and document things more inside docs/. WDYT ? Should I even wipe the README.md from builder ?

Reply:
Let's keep these README files for now. We can do a follow up to redact and publish as docs.